css.md

```diff
@@ -26,6 +26,19 @@ ITCSS organizes CSS into eight "groups" that can even be used to organize your L
 
 It's also a helpful pattern to prefix selectors (i.e. `.o-, .c-, .u-`) using these group names so that it's easy to identify the role of a selector in markup.
 
+### BEM
+
+Use the [BEM methodology](http://getbem.com/introduction/) to name class selectors.
+
+```html
+<div class="c-card c-card--theme">
+  <div class="c-card__body">
+    <h3 class="c-card__title c-card__title--editorial">Vulputate Bibendum</h3>
```

Comment:
We should avoid classes that are context-specific, so `--editorial` probably isn't the right example. Maybe `--featured` or something?

Reply:
Seems like the same thing to me, but I can adjust.